src/main/java/org/kcctl/command/ApplyCommand.java

```diff
@@ -44,7 +45,7 @@
     @Inject
     ConfigurationContext context;
 
-    @Option(names = { "-f", "--file" }, description = "Name of the file to apply", required = true)
+    @Option(names = { "-f", "--file" }, description = "Name of the file to apply or '-' to read from stdin", required = true)
```

Comment:
Could you also update the description of the command itself (and copy over the output into the README, which currently serves as the reference for all commands).